ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -1512,6 +1526,43 @@ export const WorkspaceEdit = fp.flow(
             {loading && (
               <SpinnerOverlay overrideStylesOverlay={styles.spinner} />
             )}
+            {showWorkspaceCreatedPopup && (
+              <Modal>
+                <ModalTitle>Workspace ready in Verily Workbench</ModalTitle>
+                <ModalBody>
+                  Your workspace "{this.state.workspace.name}" has been
+                  successfully created. Start exploring your new research
+                  environment.
+                  <br />
+                </ModalBody>
+
+                <ModalFooter>
+                  <Button
+                    type='secondaryLight'
+                    onClick={() => {
+                      this.setState({ showWorkspaceCreatedPopup: false });
+                      this.props.navigate(['/workspaces']);
```

Comment:
Is this navigation expected? I wonder if the user would be surprised if after clicking "Stay here" they are brought to a different screen.

Reply:
I think I'll update the text a bit, I have an updated mock to make it explicit that the user will return to the list.

Reply:
@evrii I made the change, please take a quick look when you have a minute

Reply:
That looks good to me